aeson.cabal

```diff
@@ -110,6 +110,10 @@ library
     -- `Data.Semigroup` is available in base only since GHC 8.0
     build-depends: semigroups >= 0.16.1
 
+  if !impl(ghc >= 7.10)
+    -- `Numeric.Natural` is available in base only since GHC 7.8
+    build-depends: nats >=1 && <1.1
+
```

Comment:
`impl(ghc < 7.10)` is subtly different from `!impl(ghc >= 7.10)` in its semantics

one implies `impl(ghc)` the other one does not